gopher/ontologies.py

```diff
@@ -48,8 +49,14 @@ def load_ontology():
                 term_id = val
             elif key == "name":
                 term_name = val
+            elif key == "is_a":
```

Comment:
I did consider including `part_of` relationships to the analysis. However, `part_of` relationships can be across aspects. `is_a` relationships are all within the same aspect. I decided to only include the `is_a` relationship due to this potential issue. That being said, it would be easy to add the `part_of` relationships if that is what we decide.